scripts/benchmark/checks/erc20_interact.py

```diff
@@ -0,0 +1,354 @@
+            gas=SECOND_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx1['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx1)
+
+        assert computation.is_success
+        assert to_int(computation.output) == 1
+
+    def erc_approve(self, addr2: str, chain: MiningChain) -> None:
+        ammout = self.num_blocks * self.num_tx
+
+        w3_tx2 = self.persist.simple_token.functions.approve(
+            addr2, ammout).buildTransaction(W3_TX_DEFAULTS)
+
+        tx2 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=FUNDED_ADDRESS_PRIVATE_KEY,
+            from_=FUNDED_ADDRESS,
+            to=self.persist.deployed_contract_address,
+            amount=0,
+            gas=SECOND_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx2['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx2)
+
+        assert computation.is_success
+        assert to_int(computation.output) == 1
+
+    def erc_transfer_from(self, addr1: str, addr2: str, chain: MiningChain) -> None:
+
+        w3_tx3 = self.persist.simple_token.functions.transferFrom(
+            addr1, addr2, 1).buildTransaction(W3_TX_DEFAULTS)
+
+        tx3 = new_transaction(
+            vm=chain.get_vm(),
+            private_key=SECOND_ADDRESS_PRIVATE_KEY,
+            from_=SECOND_ADDRESS,
+            to=self.persist.deployed_contract_address,
+            amount=0,
+            gas=SECOND_TX_GAS_LIMIT,
+            data=decode_hex(w3_tx3['data']),
+        )
+
+        block, receipt, computation = chain.apply_transaction(tx3)
+
+        assert computation.is_success
+        assert to_int(computation.output) == 1
+
+
+class ERC20DeployBenchmark(BaseERC20Benchmark):
+    def __init__(self, persist: Persistant) -> None:
+        super().__init__(persist)
+
+    @property
+    def name(self) -> str:
+        return 'ERC20 deployment'
+
+    def _setup_benchmark(self, chain: MiningChain) -> None:
+        return
+
+    def _do_benchmark(self, chain: MiningChain) -> None:
+        self.deploy_simple_token(chain)
+
+
+class ERC20TransferBenchmark(BaseERC20Benchmark):
+    def __init__(self, persist: Persistant) -> None:
+        super().__init__(persist)
+
+    @property
+    def name(self) -> str:
+        return 'ERC20 Transfer'
+
+    def _setup_benchmark(self, chain: MiningChain) -> None:
+        self.deploy_simple_token(chain)
+        chain.mine_block()
+        return
+
+    def _do_benchmark(self, chain: MiningChain) -> None:
+        self.erc_transfer(self.addr1, chain)
+
+
+class ERC20ApproveBenchmark(BaseERC20Benchmark):
+    def __init__(self, persist: Persistant) -> None:
+        super().__init__(persist)
+
+    @property
+    def name(self) -> str:
+        return 'ERC20 Approve'
+
+    def _setup_benchmark(self, chain: MiningChain) -> None:
+        self.deploy_simple_token(chain)
+        self.erc_transfer(self.addr1, chain)
+        chain.mine_block()
+        return
```

Comment:
No need to explicitly write `return`. Every method returns implicitly with `None` if no `return <something>` is given